h2/src/test/org/h2/test/testScript.sql

```diff
@@ -10588,4 +10588,76 @@ create table z.z (id int);
 > ok
 
 drop schema z;
-> ok
\ No newline at end of file
+> ok
+
+----------------
+--- ENUM support
+----------------
+
+--- ENUM basic operations
```

Comment:
@inikolaev I tried adding this test but, unfortunately, it did not work. I would be happy to address this in a follow-on PR.